client/extensions/woocommerce/app/products/product-form-additional-details-card.js

```diff
@@ -0,0 +1,188 @@
+		this.debouncedUpdateName( e.target.id, e.target.value );
+	}
+
+	updateName( attributeId, name ) {
+		const { product, editProductAttribute } = this.props;
+		const attribute = this.getAttribute( product, attributeId );
+		editProductAttribute( product, attribute, { name } );
+	}
+
+	renderInput( attribute ) {
+		const { translate } = this.props;
+		const { attributeNames } = this.state;
+		const attributeName = attributeNames && attributeNames[ attribute.uid ] || attribute.name;
+		return (
+			<div key={ attribute.uid } className="products__additional-details-form-fieldset">
+				<FormTextInput
+					placeholder={ translate( 'Material' ) }
+					value={ attributeName }
+					id={ attribute.uid }
+					name="type"
+					onChange={ this.updateNameHandler }
+				/>
+				<TokenField
+					placeholder={ translate( 'Cotton' ) }
+					value={ attribute.options }
+					name="values"
+					/* eslint-disable react/jsx-no-bind */
+					onChange={ values => this.updateValues( values, attribute ) }
+				/>
+				<Button
+					borderless
+					onClick={ this.removeAttribute }
+					id={ attribute.uid }
+				>
+					<Gridicon icon="cross-small" />
+				</Button>
+			</div>
+		);
+	}
+
+	renderPreview( attribute ) {
+		const { attributeNames } = this.state;
+		const attributeName = attributeNames && attributeNames[ attribute.uid ] || attribute.name;
+
+		if ( ! attributeName && ! attribute.options.length ) {
+			return ( <div key={ attribute.uid }></div> );
+		}
+
+		return (
+			<div key={ attribute.uid } className="products__additional-details-preview-row">
+				<div className="products__additional-details-preview-type">{ attributeName }</div>
+				<div>{ attribute.options.join( ', ' ) }</div>
+			</div>
+		);
+	}
+
+	render() {
+		const { translate } = this.props;
+		const attributes = this.getAttributes();
+		const inputs = attributes && attributes.length && attributes.map( this.renderInput, this ) || (
+			<FormSettingExplanation>
+				{ translate( 'Click the button below to start adding additional details about your product.' ) }
+			</FormSettingExplanation>
+		);
+		const previews = attributes && attributes.map( this.renderPreview, this );
+
+		return (
+			<FoldableCard
+				className="products__additional-details-card"
+				header={ translate( 'Add additional details' ) }
+				onOpen={ this.cardOpen }
+				clickableHeader
+			>
+				<FormSettingExplanation>
+					{ translate( 'Display additional details in a formatted list. ' +
+					'Examples when selling apparal include Material, Type, and Cut. ' +
+					'This will also allow customers to filter your store to find Women cut Hoodies in Cotton.' ) }
+				</FormSettingExplanation>
+
+				<div className="products__additional-details-container">
+					<div className="products__additional-details-form-group">
+						<div className="products__additional-details-form-labels">
+							<FormLabel>{ translate( 'Type' ) }</FormLabel>
+							<FormLabel>{ translate( 'Values' ) }</FormLabel>
```

Comment:
Hi! I've found a possible matching string that has already been translated **90** times:
`translate( 'Value' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).